UBI7/Dockerfile

```diff
@@ -7,10 +7,13 @@ WORKDIR /root
 
 RUN sed -i -e 's/os.path.exists(HOST_CONFIG_DIR)/False/g' /usr/lib64/python2.7/site-packages/rhsm/config.py
 RUN echo "{\"virt.host_type\": \"Not Applicable\", \"virt.is_guest\": \"False\"}" > /etc/rhsm/facts/custom.facts
-ADD startup.sh /tmp/
-RUN chmod +x /tmp/startup.sh
-ADD hostname-3.13-3.el7.x86_64.rpm /tmp/
-RUN yum -y localinstall /tmp/hostname-3.13-3.el7.x86_64.rpm
+
+# add and process the external resources
+ADD resources/* /tmp/
+ADD setup_scripts/* /tmp/setup_scripts/
+WORKDIR /tmp
```

Comment:
add a space between ADD and WORKDIR to keep it consistent with the other Dockerfiles.